contrib/sandbox/packages/st2-mistral/actions/transformer.py

```diff
@@ -0,0 +1,60 @@
+import json
+import os
+import six
+import yaml
+
+from st2actions.runners import pythonrunner
+from st2actions.runners.mistral import utils
+from st2client import client
+
+
+class MistralDSLTransformer(pythonrunner.Action):
```

Comment:
I'll move this to the st2-incubator.  I put this here because the mistral pack was placed here first by lakshmi.  I can't use utils.transform_definition() because reading from the Action model is different.  The action here uses the st2client whereas the one in utils uses the st2common persistence directly.